src/python/deepgnn/graph_engine/snark/converter/writers.py

```diff
@@ -77,9 +77,11 @@ def add(self, data: typing.Iterator[typing.Tuple[int, int, int, float, list]]):
                 its edges in order of dst. Each entry for a node/edge is,
                 (node_id/src, -1/dst, type, weight, [ndarray for each feature vector or None in order of feature index]).
         """
+        prev_node_id = -1
```

Comment:
Please, reduce number of use magic constants and use `None` or some other way to track previous node.